CONTRIBUTING.md

```diff
@@ -4,7 +4,9 @@ This is a guide for contributing to Thrive.
 
 __Note: if you have an idea, share it on our [community
 forums](https://community.revolutionarygamesstudio.com/)!__ and don't
```

Comment:
I think for conciseness, the second sentence can be merged here:

```suggestion
__Note: if you have an idea, share it on our [community
forums](https://community.revolutionarygamesstudio.com/) or check out and vote on our
[suggestions site](https://suggestions.revolutionarygamesstudio.com/).!__and don't
```

Reply:
I felt like it makes the sentence run on for too long as there's that "don't create an issue about it" part.

Reply:
At the same time the suggestions board message will also be included in the bolded text. Whichever is better I guess.

Reply:
I guess that bolding makes it better. I'll try making the change later (currently battling with  #3706).

Reply:
Changed this.